pokemongo_bot/cell_workers/transfer_pokemon.py

```diff
@@ -10,7 +10,16 @@
 class TransferPokemon(BaseTask):
     SUPPORTED_TASK_API_VERSION = 1
 
+    def initialize(self):
+        self.min_empty_space = self.config.get('min_empty_space', None)
+
+    def get_pokemon_slot_left(self):
+        return self.bot._player["max_pokemon_storage"] - self.bot.get_inventory_count('pokemon')
+
     def work(self):
+        if self.min_empty_space is not None and self.get_pokemon_slot_left() >= self.min_empty_space:
```

Comment:
Check `inventory.py` to learn how to check for pokemon bag space properly without making extra requests.